File: src/main/java/com/rs/java/game/area/multi/CastleWarsMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.shapes.Rectangle;

public class CastleWarsMulti extends Area {

	public CastleWarsMulti() {
		super(9520);
	}

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Castle Wars Multi";
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}
}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/CastleWarsMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class CastleWarsMulti : Area(9520) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Castle Wars Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}